CHANGES.md

```diff
@@ -7,10 +7,12 @@ The released versions correspond to PyPi releases.
 * added handling of path-like where missing
 * improved handling of `str`/`bytes` paths
 * suppress all warnings while inspecting loaded modules
-  (see [#614](../../issues/614))
+  (see [#614](../../issues/614)
 
 ### Infrastructure
 * added test dependency check (see [#608](../../issues/608))
+* skip tests failing with ASCII locale
+  (see [#623](../../issues/623)
```

Comment:
same here

Reply:
Thanks - that was copy-pasted from above, where it was already wrong...